barretenberg/cpp/src/barretenberg/vm/avm/trace/trace.cpp

```diff
@@ -3208,28 +3208,47 @@ void AvmTraceBuilder::op_pedersen_commit(uint8_t indirect,
  * @param indirect A byte encoding information about indirect/direct memory access.
  * @param src_offset An index in memory pointing to the input of the To_Radix_LE conversion.
  * @param dst_offset An index in memory pointing to the output of the To_Radix_LE conversion.
- * @param radix A strict upper bound of each converted limb, i.e., 0 <= limb < radix.
+ * @param radix_offset An index in memory pointing to the strict upper bound of each converted limb, i.e., 0 <= limb <
+ * radix.
  * @param num_limbs The number of limbs to the value into.
+ * @param output_bits Should the output be U1s instead of U8s?
  */
-void AvmTraceBuilder::op_to_radix_le(
-    uint8_t indirect, uint32_t src_offset, uint32_t dst_offset, uint32_t radix, uint32_t num_limbs)
+void AvmTraceBuilder::op_to_radix_le(uint8_t indirect,
+                                     uint32_t src_offset,
+                                     uint32_t dst_offset,
+                                     uint32_t radix_offset,
+                                     uint32_t num_limbs,
+                                     uint8_t output_bits)
```

Comment:
Is there a reason this isn't of type `bool`?

Reply:
Just would be a one-off I think. Wasn't sure if I should leave it as a byte which is what it is in the wire format for the instruction. Instruction wire format operates in bytes.

But I could certainly change this to be type bool